packages/channels/botpress-channel-web/src/views/web/messages/index.jsx

```diff
@@ -229,22 +231,22 @@ class Message extends Component {
       )
     return (
       <Linkify properties={{ target: '_blank' }}>
-        <div>{element}</div>
+        <div className="botpress__message-element-text">{element}</div>
```

Comment:
BTW why do we have this word `element` in multiple places, what does it mean? Can it be `botpress__message-text` here and similarly in other places?